src/sdgio-edit.owl

```diff
@@ -3201,21 +3201,22 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020286> "improved forecasting of impacts of climate change and targeted technology development"@en)
 ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020237> <http://purl.unep.org/sdg/SDGIO_00020286>)
 
-# Individual: <http://purl.unep.org/sdg/SDGIO_00020287> (enhanced capacity to deal with climatic risks and extremes through the achievement of mitigation and adaptation plans)
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020287> (enhancement of the capacity to deal with climatic risks and extremes through the achievement of mitigation and adaptation plans)
```

Comment:
Looking at it now, the next ambiguity arises:
Who/what is getting their capacity enhanced?

Reply:
I added this ambiguity to issue #164.